app/Helper/HelperFacade.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Helper;
+
+use Illuminate\Support\Facades\Facade;
+use App\Helper\Helper;
+
+class HelperFacade extends Facade
+{
+	protected static function getFacadeAccessor()
```

Comment:
When you can easily manipulate Laravel collection what's the use of making this HelperFacade ? Next thing you need not create this HelperFacade instead create helper file and autoload it, so that it can be called from anywhere.